src/OpenTok/Broadcast.php

```diff
@@ -35,6 +35,10 @@
 * @property boolean $isStopped
 * Whether the broadcast is stopped (true) or in progress (false).
 *
+* @property string $multiBroadcastTag
+* Whether Multiple Broadcast is switched on, which will be a unique string for each simultaneous broadcast of an ongoing session.
+* See https://tokbox.com/developer/guides/archiving/#simultaneous-archives for more information.
```

Comment:
This should be  https://tokbox.com/developer/guides/broadcast/live-streaming/#simultaneous-broadcasts.